apps/pre-award/copilot/environments/addons/form-uploads.yml

```diff
@@ -48,6 +48,23 @@ Resources:
                 "aws:SecureTransport": false
       Bucket: !Ref FormUploadsBucket
 
+  FormsUploadBucketAccessPolicy:
+     Type: AWS::IAM::ManagedPolicy
+     Properties:
+       PolicyDocument:
+         Version: '2012-10-17'
+         Statement:
+           - Sid: S3FormUploadActions
+             Effect: Allow
+             Action:
+               - s3:Get*
+               - s3:List*
+               - s3:Describe*
+	       - s3:PutObject
```

Comment:
Tabs vs Spaces. It doesn't show here, but...

![image](https://github.com/communitiesuk/funding-service-design-workflows/assets/125875751/98cb0b5c-9442-4118-b130-296df19f804e)